lib/ClangImporter/ImportType.cpp

```diff
@@ -3247,9 +3250,13 @@ ImportedType ClangImporter::Implementation::importAccessorParamsAndReturnType(
   DeclContext *origDC = importDeclContextOf(property,
                                             property->getDeclContext());
   assert(origDC);
+  auto fieldType = isGetter ? clangDecl->getReturnType() : clangDecl->getParamDecl(0)->getType();
+  ImportedType importedNSOptionsType = tryImportOptionsTypeForField(fieldType, *this);
 
   // Import the property type, independent of what kind of accessor this is.
   auto importedType = importPropertyType(property, isFromSystemModule);
```

Comment:
We don't want to import the type if we have already found something with `tryImportOptionsTypeForField`

Reply:
Ah yeah good point. I think I was trying to leave the subsequent parameter-list creation code in tact here, but I see what you mean.